docs/workshop-schedule.md

```diff
@@ -3,39 +3,49 @@ title: Workshop Schedule
+| Time                | Topic                                                                                                                                                         |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 8:30 am             | _Participants arrive, have breakfast, and check in_                                                                                                           |
+| 9:00 - 10:15 am     | [Welcome and introductions](https://docs.google.com/presentation/d/1-mPY76_AbS8rLqxoSAOc2Ews5GyPSo8PVuG2El_S6BE/edit?usp=sharing)                             |
+|                     | [Why does reproducibility matter?](https://docs.google.com/presentation/d/1qfulAR4jD0KS7NfrLHpwT6SWl-7APBmqNnAwGpXX5oo/edit?usp=sharing)                      |
+| 10:15 - 10:45 am    | [Project organization](https://docs.google.com/presentation/d/1ncqxXlC0-PGEK-yE7S-nDYnMPhrOUPbI95EJy283wCs/edit?usp=sharing)                                  |
+| 10:45 - 11:00 am    | _Coffee break_                                                                                                                                                |
+| 11:00 - 11:30 am    | [Strategies for data sharing](https://docs.google.com/presentation/d/10NEkqHkAw9KBAbkNV-vtPwLFc9ZHksHEcCFty32efiE/edit?usp=sharing)                           |
+| 11:30 am - 12:00 pm | [Introduction to UNIX and the command line](https://docs.google.com/presentation/d/1WPXkItJZEUXMY20cLrdMXHiBC2PyunR14RUSDg4nfIc/edit?usp=sharing)             |
+|                     | [_Resource_: UNIX Reference](resources/unix_reference.html)                                                                                                   |
+|                     | [_Resource_: UNIX Cheatsheet (pdf)](resources/unix_quick_reference.pdf)                                                                                       |
+| 12:00 - 12:30 pm    | [Introduction to Git; Forking and cloning a repository](https://docs.google.com/presentation/d/1eiGZA4PYBKJx5HDCo3UDOAB7q415gg96TehxilCHwlA/edit?usp=sharing) |
+| 12:30 - 1:30 pm     | _Lunch_                                                                                                                                                       |
+| 1:30 - 2:15 pm      | [Shell scripting](https://docs.google.com/presentation/d/1SDUyYVNgvDDRodVqmDQPVQ5wnjQesWfBTg0EAcdbcSo/edit?usp=sharing)                                       |
+| 2:15 - 2:45 pm      | [The Git stage/commit/push workflow](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing)                    |
+| 2:45 - 3:00 pm      | _Coffee break_                                                                                                                                                |
```

Comment:
Just a note that they make me bring the coffee back no later than 3. But I think this should be fine. I'll just make people aware!